website/versioned_docs/version-29.0/ExpectAPI.md

```diff
@@ -37,52 +41,166 @@ The argument to `expect` should be the value that your code produces, and any ar
 
 You can use `expect.extend` to add your own matchers to Jest. For example, let's say that you're testing a number utility library and you're frequently asserting that numbers appear within particular ranges of other numbers. You could abstract that into a `toBeWithinRange` matcher:
 
-```js
+```js tab={"span":3} title="toBeWithinRange.js"
```

Comment:
The redundant note was not included in this file. Also the instructions were outdated. So I made all 29.x docs look the same.